src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -963,7 +1088,7 @@ def get_image_features(
         output_attentions: Optional[bool] = None,
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
-        return_projected: Optional[bool] = True,
+        return_base_image_embeds: Optional[bool] = None,
```

Comment:
This argument can be removed, see comment below

Reply:
@NielsRogge I left it for backward compatibility purposes, shall I remove it or add a deprecation warning?

Reply:
I see you're adding a new argument "return_base_image_embeds" here, so I don't see how this is for backward compatibility purposes.

Reply:
Oh yes, you can do that indeed.